### ISSUE TITLE
[Bug][a11y]: Swatches when disabled, are missing border and the fill color in WHCM

### ISSUE THREAD

**Issue by dineshy87** [adobe/spectrum-web-components#3295]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

sp-swatch

### Expected behavior

Disabled variant should have the border in HCM. It must also have the fill color in order to understand which color is disabled.

![Screenshot 2023-06-07 at 1 47 54 PM](https://github.com/adobe/spectrum-web-components/assets/16927765/bf84b791-f73a-4f90-aeff-79b5706ad915)

### Actual behavior

Missing border and fill color.

![Screenshot 2023-06-07 at 1 46 01 PM](https://github.com/adobe/spectrum-web-components/assets/16927765/e14dac02-d672-4422-ae7a-ef5268f643d9)

### How can we reproduce this issue?

1. Go to 'https://studio.webcomponents.dev/preview/JZn9nn7uzp4NE205enzB/stories?p=stories&busid=253eacf0-056e-11ee-99f2-b9caaae313c8'
2. Enable High contrast mode

**Comment by Rajdeepc** [comment]
@najikahalsema Do you know the Bug Id for this? Want to check if something that needs to be done from the SWC side or it is a change from the spectrum-css Swatch component.

**Comment by najikahalsema** [comment]
@Rajdeepc Did I answer this in our meeting yesterday? 😅 Sorry I just got back to this, my email wasn't updating.

**Comment by Rajdeepc** [comment]
Related Jira Ticket: https://jira.corp.adobe.com/browse/CSS-535

**Comment by pfulton** [comment]
Spectrum CSS fix released in: `@spectrum-css/swatch@4.0.26`

**Comment by Rajdeepc** [comment]
@dineshy87 you should be able to test this and let us know the outcome.
cc @jnjosh

**Comment by dineshy87** [comment]
Checked here:  https://pr-1955--spectrum-css.netlify.app/swatch.html#disabled 

The fill color is now visible, but border color still doesn't show in 'GrayText' as the Expected behavior screenshot shows.

<img width="1214" alt="image" src="https://github.com/adobe/spectrum-web-components/assets/16927765/e906e2ee-015e-4a12-b70d-ff7d75e4ad0b">

**Comment by pfulton** [comment]
@dineshy87 @Rajdeepc 

A little difficult to see, but does this look like the appropriate change? This is in my local environment.

<img width="1079" alt="Screenshot 2023-06-28 at 10 56 52 AM" src="https://github.com/adobe/spectrum-web-components/assets/360251/1a47ad60-96b9-4539-9b74-eb5a4b913fff">

**Comment by Rajdeepc** [comment]
Yes the border is visible.

**Comment by dineshy87** [comment]
Yeah, that works. Depending on the fill color it may be difficult to perceive, but the symbol in the middle always helps. So, this is good.

**Comment by pfulton** [comment]
PR is open here: https://github.com/adobe/spectrum-css/pull/1980

**Comment by pfulton** [comment]
This has been released and is available in: `@spectrum-css/swatch@4.0.27`

**Comment by Rajdeepc** [comment]
@dineshy87 Let us know if we can close this

**Comment by dineshy87** [comment]
> @dineshy87 Let us know if we can close this

Yes, can be closed.

**Comment by Rajdeepc** [comment]
Thanks. Closing as it is verified by the user.
